java/ql/src/Security/CWE/CWE-113/ResponseSplitting.qhelp

```diff
@@ -33,6 +41,13 @@ The second way will verify the parameters before using them to build the HTTP re
 <sample src="NettyResponseSplitting.java" />
 </example>
 
+<example>
+<p>The following example shows the use of the library 'netty' with HTTP request-splitting verification configurations.
+The second way will verify the parameters before using them to build the HTTP request.</p>
```

Comment:
I think this may be clearer if the information is inline in the code example, but if you keep it here, this may make it a bit easier to understand.

```suggestion
The second recommended approach in the example verifies the parameters before using them to build the HTTP request.</p>
```